nixnet/database/_frame.py

```diff
@@ -2,15 +2,18 @@
 from __future__ import division
 from __future__ import print_function
 
+import operator
 import typing  # NOQA: F401
 
 from nixnet import _cconsts
 from nixnet import _errors
 from nixnet import _props
 from nixnet import constants
+from nixnet import types
 
 from nixnet.database import _collection
 from nixnet.database import _dbc_attributes
+from nixnet.database import _pdu
```

Comment:
This import will need to be moved down into the methods that use PDUs, else we will get circular imports when combined with #237.

Reply:
I fixed this circular important but ran into another one trying to import nixnet.database._pdu in types.py because the PduProperties class has a Pdu object as one of its values and I wanted to reference it in the documentation. Why are we importing stuff for sphinx in every file instead of adding our whole library to the path in sphinx's conf.py?

Reply:
> Why are we importing stuff for sphinx in every file instead of adding our whole library to the path in sphinx's conf.py?

I don't follow. What imports are you talking about?

Reply:
Damon and I talked about this offline and I found out I could avoid importing a class for sphinx to see it by not using the fully qualified name.